docs/reference/settings/health-api-settings.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[health-api-settings]]
+=== Health API settings in {es}
+[subs="attributes"]
+++++
+<titleabbrev>Health API settings</titleabbrev>
+++++
+
+The following are the _expert-level_ settings available for configuring the
+<<health-api, Health API>>. It is not recommended to change any of these
```

Comment:
I'd say this configures an internal diagnostics service as opposed to the API. 

What do you think?

Reply:
I like this idea. Essentially these are internal diagnostics, they are now used in the health API but they might have more use cases, right? 

What about calling this page "Health diagnostic settings" or "Internal health diagnostic settings" and only refer to the health API in the begging. Something like: "the outcome of this service is exposed via the Health API"?

I think this could open up different ways of phrasing the impact of the settings. For example: 

`This many changes or more will be reported as a problem in the Health API` could be phrased as `This many changes or more will be reported as unhealthy`?

Reply:
++ I like it too and I like your suggestions @gmarouli !